ffmpeg/lpms_ffmpeg.c

```diff
@@ -751,14 +751,14 @@ int process_out(struct input_ctx *ictx, struct output_ctx *octx, AVCodecContext
 
 #define MAX_OUTPUT_SIZE 10
 
-int lpms_transcode(input_params *inp, output_params *params, int nb_outputs)
+int lpms_transcode_and_count(input_params *inp, output_params *params, int nb_outputs, int *framecount, int *pixelcount)
```

Comment:
We cannot keep the original lpms_transcode name because this is `cgo` , which is actually the `c` language.  `C` does not support overloading function names as is supported by `c++`

So, unfortunately, the name of the function must change here.

Reply:
We aren't overloading anything, just changing the function signature. Retaining back compat for this internal API is not necessary.